pkg/config/config.go

```diff
@@ -81,14 +86,74 @@ func (c *Config) ConfigureRules() {
 }
 
 func loadConfig(filename string) (c Config, err error) {
+	if isValidURL(filename) {
+		// if fileName is a valid URL, we will download and set it to the config
+		log.Debug().Str("url", filename).Msg("Downloading file from")
+		// hardcoding this file and saving to root directory
+		downloadedFile := "downloadedRules.yaml"
+		err := DownloadFile(downloadedFile, filename)
+		if err != nil {
+			return c, err
+		}
+		filename = downloadedFile
+		log.Debug().Str("filename", filename).Msg("Saved remote config to local file.")
+	}
+
+	// TO DO = Need to add more error handling on reading/validating yamlFile file - issue # 16
```

Comment:
I think this is fine keeping the TODO item here for visibility, but keep in mind referencing issue #16 upstream won't make sense.

Reply:
good call - i may just remove for now